app/views/RoomsListView/index.js

```diff
@@ -434,6 +439,8 @@ class RoomsListView extends React.Component {
 				onChangeSearchText={this.search}
 				toggleSort={this.toggleSort}
 				goDirectory={this.goDirectory}
+				onCancelPress={this.onCancelPress}
```

Comment:
why? you can use `this.cancelSearching` here and not need create `onCancelPress`